tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1166,7 +1166,7 @@ qos/test_buffer.py::test_buffer_model_test:
   skip:
     reason: "Running only on mellanox devices and covered by unit testing"
     conditions:
-      - "asic_type in ['mellanox']"
+      - "asic_type in ['mellanox'] or platform in ['x86_64-nokia_ixr7250e_36x400g-r0']"
```

Comment:
is this an asic limitation? can we add asic_subtype == 'dnx' iso of platform check?